app/views/devise/registrations/edit.haml

```diff
@@ -75,194 +230,25 @@
-                                    .form-group
-                                        = f.submit "Update", :class => "btn btn-primary"
-                    .tab-pane.fade#premium
-                        %h3.strong Premium Features
-                        %hr
-                        .row
-                            = form_for(resource, :as => resource_name, :url => registration_path(resource_name), :html => { :method => :put }) do |f|
-                                .col-md-6
-                                    .form-group{style: "margin-top: 16px;"}
-                                        = f.label :default_server_id, 'Default Server'
-                                        = f.select :default_server_id, [nil, *Server.bukkits.searchable.sort_by(&:name).map{|s| [s.name, s.id]}], { :model => Server }, :class => 'form-control', :disabled => !@user.can_set_default_server?
-                                    .form-group{style: "margin-top: 16px;"}
-                                        = f.label :death_screen, 'Death Screen'
-                                        = f.select :death_screen, User.death_screens.invert, { :model => String }, :class => 'form-control', :disabled => !@user.can_set_death_screen?
-                                .col-md-6
-                                    .checkbox
-                                        = f.label :beta_participant do
-                                            = f.check_box :beta_participant, :disabled => (@disabled.include? :beta_participant)
-                                            Beta Participation
-                                .col-md-12
-                                    .form-group
-                                        = f.submit "Update", :class => "btn btn-primary"
+                                            Enable Dyslexia Assist
+                                .form-group{style: "margin-top: 16px;"}
+                                    = f.label :time_zone_name, 'Time Zone'
+                                    = f.time_zone_select :time_zone_name, TZInfo::Timezone.all.sort, { :model => TZInfo::Timezone, :disabled => (@disabled.include? :time_zone_name) }, :class => 'form-control'
+                            .col-md-12
+                                .form-group
+                                    = f.submit "Update", :class => "btn btn-primary"
+                - if current_user.can_edit_verified_profile?
                     .tab-pane.fade#verified
                         %h3.strong Verified Information
                         %hr
                         .row
                             .col-md-12
                                 %p
                                     These fields affect site functionality in various ways, and are trusted to be accurate.
-                                    Only certain staff members have access to these.
+                                    Only certain staff members have access these.
```

Comment:
Refering to this PR (#24), seems like you're reverting the change made. However, by doing so the grammar won't be correct.

Reply:
Updates